src/ctrl/PopupMenuParamWidget.h

```diff
@@ -16,48 +16,78 @@
 
 class PopupMenuParamWidget : public ::rack::app::ParamWidget {
 public:
-    std::vector<std::string> labels;
```

Comment:
labels -> longLabels and shortLabels. If you don't use the short ones, nothing is changes. Of course no code in this repo uses the short labels.